packages/framework/client-logger/external-telemetry/src/common/consumers/appInsightsTelemetryConsumer.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+import type { ITelemetryConsumer } from "./index.js";
+import type { IExternalTelemetry } from "../telemetry/index.js";
+
+/**
+ * A simple implementation of {@link ITelemetryConsumer} for sending {@link IExternalTelemetry}
+ * to Azure App Insights in a controlled manner.
+ *
+ * @internal
+ */
+export class AppInsightsTelemetryConsumer implements ITelemetryConsumer {
```

Comment:
I wonder if this is something we need to expose. It seems like we could provide this as just a example in documentation form, and let consumers implement it. 

Seems like this package shouldn't _really_ need to know anything about App Insights. We could just provide documented examples of plugging our tiny API into external systems.

Reply:
Yeah, we discussed the option, I initially thought this was fine... but on second thought I tend to agree.

Reply:
Updated to remove app insights. We just need to make sure it is very clear to customers what we expect the consumer.consume() method to look like.